build.gradle

```diff
@@ -98,7 +118,7 @@ List<String> resolveIntelliJVersions(String versionKey) {
   }
 
   def versionValue = project.intellijVersions.get(versionKey)
-  if (versionValue == null) {
+  if (!versionValue) {
```

Comment:
Actually... am I right that this method is actually only used for `resolveIntelliJVersions` task, which itself is only used in a single script under `scripts/`? 🤔 I think now that versions are stored in properties, that script can just rely on `grep` over properties... and this method/task can be removed

Reply:
yes task can be removed (and the script needs to be changed), but this method resolveIntelliJVersions is also used in resolving ui test targets (line 658)